index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>TODO supply a title</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+        <script type="text/javascript" src="js/sheet.js"></script>
+        <link rel="stylesheet" href="css/timesheet.css">
+
+        <style>
+            body{
+                background: #6A6868;
+            }
+
+            *{
+                padding: 0;
+                margin: 0;
+            }
+
+        </style>
+    </head>
+    <body>
+        <div class="main-wrapper">
+            <h2>Timesheet Chart</h2>
+            <canvas id="mycanvas" width='1220' height='500'>
+            </canvas>
+        </div>
+        <script type="text/javascript">
+            $(document).ready(function() {
```

Comment:
iife seems to be missing and i think this code should be moved to seperate js file